src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, Area {

```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line before the class declaration.